### PR TITLE
Documentation changes.

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**
I stumbled over a few things when setting this package up,
and I thought I'd add some documentation to keep others from
hitting those sanme issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ and store the credentials for the Identity and Aria azure devops instances.
 1. Go to //myaccess
 2. Send a request to join "Identity Apps Team - 18174"
 3. Install [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) or [Git Credential Manager for Mac and Linux](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux) and setup with your MSFT credential.
+4. Go to github.com.  In the top let corner of the screen you'll create you'll see an icon or a profile picture.  Click it, select '''Settings'''.  On the subsequent screen, select '''Developer Settings''', and '''Personal Access Tokens''' on the subsequent screen.  Select Generate New Token, and give it permission for repo. Make a note of this token, because you'll need it later.
```
Comment (shoatman) [inline README.md:21]: I don't think this is necessary if you use the credential manager for Windows.  Are you on MAC or Windows?

Comment (AdamBJohnsonx) [reply README.md:21]: OK, this is probably an iOS distinction then.

Comment (rpdome) [reply README.md]: I used SSH key on OSX. https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh

Comment (rpdome) [reply README.md]: Just realized that this step (setting up GitHub creds) is totally missing. The [Git-Credential-Manager-for-Windows] step above is specificially for MSFT account (AuthApp is hosted inside Azure DevOps) I think.

Comment (rpdome) [reply README.md]: Added this as part of the new 'Private GitHub Repositories' section in my PR https://github.com/AzureAD/android-complete/pull/32/commits/c8b7f49cb408d5b674f49ce7a2349e59104892a0

Comment (AdamBJohnsonx) [reply README.md]: Cool.  I can drop this in favor of yours.

**Pull request closed without merging**